### ISSUE TITLE
FBT003 false positive when using namedtuple

### ISSUE THREAD

**Issue by AaronDMarasco** [astral-sh/ruff#10915]
It seems to be confused and think that the naked booleans are parameters to a call.

* A minimal code snippet that reproduces the bug.

```python
from collections import namedtuple

MyType = namedtuple('MyType', 'flag1, flag2, flag3')

x = MyType(True, False, True)
```

* The command you invoked (e.g., `ruff /path/to/file.py --fix`), ideally including the `--isolated` flag.

```sh
$ ruff check --isolated --config "lint.select=['FBT003']" FBT003.py 
FBT003.py:5:12: FBT003 Boolean positional value in function call
FBT003.py:5:18: FBT003 Boolean positional value in function call
FBT003.py:5:25: FBT003 Boolean positional value in function call
Found 3 errors.
```

* The current Ruff settings (any relevant sections from your `pyproject.toml`).
  * N/A
  
* The current Ruff version (`ruff --version`).
  * ruff 0.3.7

**Comment by charliermarsh** [comment]
I think the spirit of the rule is such that you should be using keyword arguments for this. Otherwise, it's easy to mix up `flag1` vs. `flag2` vs. `flag3`.

The rule is very strict and you may not want to enable it, but this seems well within its bounds.

**Comment by AaronDMarasco** [comment]
I understand the rule, it just seems silly to be applied to `namedtuple`. Yeah, _technically_ you're calling a constructor to an object, but not the in the way most people would think (IMHO).
